CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 ``
 ## UNRELEASED
+## [3.0.0] - 2021-12-29
+
+- First version of chowbus. Added multiple clusters support for producers.
```

Comment:
If we intend to have our charges merged upstream, it's best to leave out the company text.

Reply:
i don't think we will do so, and they definitely won't agree these changes...